examples/user-management/ionic-vue-user-management/.env.example

```
@@ -0,0 +1,2 @@
+VUE_APP_SUPABASE_URL=https://moijyfpvgnmgoxvwcikq.supabase.co
+VUE_APP_SUPABASE_PUBLISHABLE_KEY=your-publishable-key-from-dashboard
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VUE_APP_SUPABASE_PUBLISHABLE_KEY key should go before the VUE_APP_SUPABASE_URL key

(UnorderedKey)